hammerdb/hammerdb

```diff
@@ -319,9 +319,9 @@ rm -f results_pbench.tar
 
 tmp_file=`mktemp /tmp/hammer_data.XXXXX`
 find -L $test  -type f  -exec grep -Iq . {} \; -print > $tmp_file
-echo "/usr/local/HammerDB/results_hammerdb_${test}.csv" >> $tmp_file
+#echo "/usr/local/${test}/results_hammerdb_*.csv" >> $tmp_file
```

Comment:
We should really look into trying to find a logger for this work.  The command `logger` _might_ work, but that logs to the system logger and I'm not sure if that's what we want to do.

No changes needed